### PR TITLE
reduce number of timeouts; make timeout more stable

### PR DESCRIPTION
**Pull request by chaosma**

### CODE REVIEW

File: consensus/config.go [hunk 1]
```diff
@@ -11,13 +11,15 @@ const (
...
 	maxLogSize    uint32        = 1000
 )
 
+type timeoutType int
+
 const (
-	timeoutNormal     = "normal"
-	timeoutViewChange = "viewchange"
+	timeoutNormal timeoutType = iota
+	timeoutViewChange
```
Comment (rlan35) [inline consensus/config.go]: const should be have first letter upper case.

File: consensus/config.go [hunk 2]
```diff
@@ -11,13 +11,15 @@ const (
...
 	// timeout duration for announce/prepare/commit
-	phaseDuration time.Duration = 20 * time.Second
+	phaseDuration time.Duration = 60 * time.Second
 	maxLogSize    uint32        = 1000
 )
 
+type timeoutType int
+
 const (
-	timeoutNormal     = "normal"
```
Comment (rlan35) [inline consensus/config.go]: I think change "Normal" to "Consensus" will be more informative.

File: consensus/config.go [hunk 3]
```diff
@@ -11,13 +11,15 @@ const (
 	viewChangeDuration time.Duration = 30 * time.Second
 
 	// timeout duration for announce/prepare/commit
-	phaseDuration time.Duration = 20 * time.Second
+	phaseDuration time.Duration = 60 * time.Second
 	maxLogSize    uint32        = 1000
 )
 
+type timeoutType int
```
Comment (rlan35) [inline consensus/config.go]: TimeoutType

Comment (chaosma) [reply consensus/config.go]: why upper case? we only use it internally

File: consensus/config.go (hunk 1 above)
Comment (chaosma) [reply consensus/config.go]: Okay

File: consensus/config.go (hunk 2 above)
Comment (chaosma) [reply consensus/config.go]: OK

**Pull request merged**